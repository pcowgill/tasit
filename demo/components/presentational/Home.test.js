import React from "react";
import { shallow } from "enzyme";
import Home from "./Home";

describe("Home", () => {
  it("renders the component", async () => {
    expect(shallow(<Home onPress={() => {}} />)).toMatchSnapshot();
  });
});
