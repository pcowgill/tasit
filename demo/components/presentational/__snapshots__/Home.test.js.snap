// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders the component 1`] = `
<Component
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Component
    source={1}
  />
  <LargeText>
    Tasit
  </LargeText>
  <Button
    onPress={[Function]}
    title="Decentraland"
  />
</Component>
`;
